pkg/scrape/scraper.go

```diff
@@ -79,6 +82,58 @@ func NewPromScraper(scrapeURL string, logger log.Logger, opts ...ScraperOption)
 }
 
 func (ps *PromScraper) Scrape() (*Result, error) {
+	if ps.scrapeFile != "" {
+		return ps.ScrapeFile()
+	}
+
+	return ps.ScrapeHTTP()
+}
+
+func (ps *PromScraper) ScrapeFile() (*Result, error) {
```

Comment:
these methods are used only internally, they should have been private.
```suggestion
func (ps *PromScraper) scrapeFile() (*Result, error) {
```

Reply:
Happy to make the method private, but the struct already has a field `scrapeFile` containing the path to the file, so I'll need to rename it to avoid the conflict

https://github.com/pedro-stanaka/prom-scrape-analyzer/pull/37/files#diff-911d403369d78bdcf12f134b868ff2698275276c4450cca0a79a57b4a4f8a149R27